app/models/manageiq/providers/embedded_ansible/automation_manager/job.rb

```diff
@@ -3,8 +3,100 @@ class ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Job < ManageIQ::P
 
   require_nested :Status
 
+  def self.run_playbook(task_id)
+    task = MiqTask.find(task_id)
+    options = task.context_data
+    name = build_name(options[:name])
+    description = options[:description]
+    info = options[:config_info]
+    playbook = ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Playbook.find(info[:playbook_id])
```

Comment:
@syncrou 
should we have a tower function that returns the tower

``` def tower(info)
          ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Playbook.find(info[:playbook_id]).try(:manager)
end
```

Since its gets used in multiple places